parlai/chat_service/core/agents.py

```diff
@@ -10,7 +10,7 @@
 from parlai.core.agents import Agent
 
 
-class ChatServiceAgent(ABC, Agent):
+class ChatServiceAgent(Agent, ABC):
```

Comment:
is this convention? or an actual bug?

Reply:
I did this as convention since that order determines the priority of inherited methods (not that it'll actually make a difference right now)